documentation/src/main/asciidoc/reference/mapping-indexedembedded.asciidoc

```diff
@@ -298,8 +314,16 @@ constraints only need to match the nearest static parent of a dynamic field
 in order for that field to be included.
 ====
 
-Below are two examples: one leveraging `includePaths` only,
-and one leveraging `includePaths` and `includeDepth`.
+[NOTE]
+.Mixing `includePaths` and `excludePaths` at different nesting levels
+====
+In general, it is possible to use `includePaths` and `excludePaths` at different levels of nested `@IndexedEmbedded`.
+When doing so, keep in mind that the resulting filters cannot be conflicting, i.e. including a property at one level
+and excluding it at the other will result in an exception being thrown.
```

Comment:
Ouch 😆  but that was the whole point of adding that `findConflictingIncludeExcludePaths` stuff ... 
What I was trying to address there was something like:
```java
class IndexedEntity {
	@IndexedEmbedded(excludePaths = "subIncluded.subIncludedString")
	Included included;
}

class Included {
	@KeywordField
	String includedString;
	// but we've excluded it explicitly already at the very top with exclude filter
	// should we include? exclude? no idea let's fail instead :)
	@IndexedEmbedded(includePaths = "subIncludedString") 
	SubIncluded subIncluded;
}

class SubIncluded {
	@KeywordField
	String subIncludedString;
}
```
and what I was thinking that we cannot simply leave it to path tracking or what we had before, since for example we could have something included/excluded because of a depth, but if it is included by depth and excluded by a excludePaths entry - it's ok we should allow it...

Reply:
Discussed here: https://hibernate.zulipchat.com/#narrow/stream/132092-hibernate-search-dev/topic/HSEARCH-1182.20IndexEmbedded.20exclude.20paths